build.gradle.kts

```diff
@@ -28,4 +28,15 @@ project(":sonar-helm-for-iac") {
     }
 }
 
+listOf(":its:plugin", ":its:ruling").forEach { path ->
+    project(path) {
+        sonar {
+            properties {
+                property("sonar.sources", "")
+                property("sonar.tests", "src/integrationTest")
```

Comment:
This was an attempt to resolve `Assertions should not be used in production code` constantly raised over ITs code. However, I don't see a result from it in SC..